e2e/tests/assorted/02-broadcast.spec.js

```diff
@@ -28,8 +28,9 @@ describe('Broadcast room', () => {
 		await sleep(1000);
 		await waitFor(element(by.id('create-channel-view'))).toExist().withTimeout(5000);
 		await element(by.id('create-channel-name')).replaceText(`broadcast${ data.random }`);
-		await sleep(1000);
-		await element(by.id('create-channel-broadcast')).tap();
+		await sleep(2000);
+		await element(by.id('create-channel-broadcast')).tap(); //FLAKY
+		await expect(element(by.id('create-channel-broadcast'))).toHaveValue('1');
```

Comment:
There's a dedicated `expect` in here since the action in the line above fails for me in about 10% of runs. (This is flipping the "broadcast" toggle for the channel whilst creating it)

I dedicated some time to fixing it, and failed, so resorted to failing earlier and making the reason more obvious.

Reply:
It's ok. Detox (and most of the e2e testing libs as well) is sadly not completely reliable.